oaknational/Oak-Web-Application
# wip: replacing homepage buttons with links

### Job Story

**When** I go to any landing page,
**I want to** make sure they are optimised for SEO
**So that we can** retain our ranking position and sitelinks.


## How to test

1. Go to https://deploy-preview-2759--oak-web-application.netlify.thenational.academy
2. You should see the that the 4 tab navigations are rendered as links

## Screenshots

How it used to look (delete if n/a):
![image](https://github.com/user-attachments/assets/b721ffe3-3ba2-43c3-9d4b-44e60035def3)


How it should now look:
![image](https://github.com/user-attachments/assets/51910b0b-d77f-4900-8821-2813b3ad6bec)


## Checklist

- [x] Buttons are links
- [x] Navigation works correctly
- [x] Styling preserved
- [x] keyboard navigation and screenreader
